src/main/kotlin/no/nav/syfo/behandler/fastlege/FastlegeClient.kt

```diff
@@ -49,7 +50,28 @@ class FastlegeClient(
             personident = personident,
             token = newToken,
             url = url,
-        )
+        ).also {
+            COUNT_CALL_FASTLEGEREST_FASTLEGE_SUCCESS.increment()
+        }
+    }
+
+    suspend fun fastlegevikar(
```

Comment:
Jeg er jo egentlig litt fan av at funksjoner har en slags verb-prefix, så `getFastlegevikar` og `getFastlege` etc. For når den brukes i koden nå ser det mer ut som et felt/en variabel enn en funksjon som gjør noe.

Men vet at vi har gjort det på denne måten for mange av API-kallene som går ut, det er feks mye `identer()` eller `person()` når man henter fra PDL rundt om i kodebasene våre.

Reply:
Ja, tror det er gjort litt forskjellig som du påpeker. I dette konkrete tilfellet forsøker jeg å følge mønsteret fra den funksjonen som var der fra før som bare heter "fastlege".